core/src/components/range/test/label/range.e2e.ts

```diff
@@ -7,7 +7,7 @@ configs().forEach(({ title, screenshot, config }) => {
       test('should render a range with no visible label', async ({ page }) => {
         await page.setContent(
           `
-          <ion-range legacy="true"></ion-range>
+          <ion-range style="padding-inline-start: 20px;" aria-label="Range"></ion-range>
```

Comment:
Why was padding added here?

Reply:
The modern range doesn't add any padding, so the knob will get cut off in this position. This is the same thing we do in other tests: https://github.com/ionic-team/ionic-framework/blob/88de029483dcadcfb61c6b4369483fff69eca0b6/core/src/components/range/test/label/range.e2e.ts#L64

Reply:
This is because the screenshot will cut off some of the knob without the padding. This is because renders outside of the element when a label isn't in front of it. So Playwright only takes a screenshot of the actual element and doesn't realize that knob is outside.

![Screenshot 2024-01-29 at 11 55 19 AM](https://github.com/ionic-team/ionic-framework/assets/13530427/2780121c-3f2e-4a8c-ad1a-630364c739ce)

Without padding:
![range-no-items-no-label-ios-ltr-actual](https://github.com/ionic-team/ionic-framework/assets/13530427/5f6d9ed8-c6bf-412d-8fd5-160b72d251d2)

With padding:
![range-no-items-no-label-ios-ltr-expected](https://github.com/ionic-team/ionic-framework/assets/13530427/d2a13c08-d03a-4965-9f12-16b3bff05211)